scrapper.py

```diff
@@ -44,18 +75,50 @@ def get_search_urls(self):
         pass
 
 
+class ArticleParser:
```

Comment:
where from do you get this name? documentation states the HTMLParser

Reply:
done